pkgs/by-name/sc/scd2html/package.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchFromSourcehut
+, buildPackages
+, scdoc
+}:
+
+stdenv.mkDerivation rec {
+  pname = "scd2html";
+  version = "1.0.0";
+
+  src = fetchFromSourcehut {
+    owner = "~bitfehler";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-oZSHv5n/WOrvy77tC94Z8pYugLpHkcv7U1PrzR+8fHM=";
+  };
+
+  strictDeps = true;
+
+  nativeBuildInputs = [
+    scdoc
+  ];
+
+  postPatch = ''
+    substituteInPlace Makefile \
+      --replace "-static" ""
+  '';
+
+  makeFlags = [
+    "PREFIX=${placeholder "out"}"
+  ];
+
+  doCheck = true;
```

Comment:
From logs:

```
running tests
check flags: SHELL=/nix/store/xdqlrixlspkks50m9b0mpvag65m3pf2w-bash-5.2-p15/bin/bash PREFIX=/nix/store/a5xry101c2qbsg4z567x40d3kf3hyygv-scd2html-1.0.0 VERBOSE=y check
make: Nothing to be done for 'check'.
```

This check phase is inexpensive, so we could leave it for the sake of future proofing, but it advertises the wrong behaviour. I would appreciate a comment